setup.py

```diff
@@ -2,19 +2,20 @@
 
 
 setup(name='mlaut',
-      version='0.1.14',
+      version='0.1.2',
```

Comment:
0.1.2 is lower than 0.1.14 no? I'd make it 0.2.0

Reply:
ok